api/envoy/api/v2/core/health_check.proto

```diff
@@ -33,9 +33,14 @@ message HealthCheck {
   ];
 
   // An optional jitter amount in millseconds. If specified, during every
-  // internal Envoy will add 0 to interval_jitter to the wait time.
+  // interval Envoy will add 0 to interval_jitter to the wait time.
   google.protobuf.Duration interval_jitter = 3;
 
+  // An optional jitter amount as a percentage of interval_ms. If specified,
+  // during every interval Envoy will add 0 to interval_ms *
+  // interval_jitter_percent / 100 to the wait time.
```

Comment:
Please document what happens if both this and interval_jitter are set. Or only allow one to be set and document that (and validate that somewhere).

Reply:
done!